V2/gameFuseUser.js

```diff
@@ -1,8 +1,7 @@
 ﻿// V2
 class GameFuseUser {
     constructor(signedIn = false, numberOfLogins = 0, lastLogin = undefined, authenticationToken = "",
-        username = "", score = 0, credits = 0, id = 0, attributes = {}, purchasedStoreItems = [],
-        leaderboardEntries = [], isOtherUser = false) {

```

Comment:
Got rid of some attributes that we aren't passing in anywhere - they all get added after the user object is already instantiated.